apps/maestro/src/lib/utils/gform.ts

```diff
@@ -0,0 +1,43 @@
+import {
+  ClaimOwnershipFormFieldIds,
+  NEXT_PUBLIC_CLAIM_OWNERSHIP_FORM_LINK,
+} from "~/config/env";
+
+const chainIdToGFormChainId = {
```

Comment:
is there another way to capture this mapping other than maintaining yet another hard-coded list?

Reply:
I think to achieve dynamically-defined values will be too over-engineered for this feature. Also, i can't find how to query all available options for multiple-choices field.